std_msgs/Header header
int64 code
int32 id
int32 hammingDistance
Point2D_32[4] corners
Point2D_32 center
